NeoML/docs/ru/API/NN/QualityControlLayers/AccuracyLayer.md

```diff
@@ -37,10 +37,10 @@ void SetReset( bool value );
 - `Height`, `Width` и `Depth` равны `1`;
 - `Channels` равен `1` для бинарной классификации или числу классов (больше `1`) для многоклассовой.
 
-На второй вход подается блоб с правильной разметкой, имеющий те же размеры, что и блоб первого входа. 
+На второй вход подается блоб с правильной разметкой, имеющий те же размеры или в случае кроссэнтропии допустимо передавать в канале метку: `Channels` = 1
 
 - если `Channels` у блоба первого входа равен `1`, правильная разметка должна содержать `1` для объектов одного класса и `-1` для другого;
-- если `Channels` больше `1`, правильная разметка должна содержать `1` для правильного класса и `0` для всех остальных.
+- если `Channels` больше `1`, правильная разметка должна содержать `1` для правильного класса и `0` для всех остальных или если `Channels` разметки равен 1, то метки правильного класса.
```

Comment:
The whole description of the second input should be rewritten in a way similar to the label input of [CCrossEntropyLayer](https://github.com/neoml-lib/neoml/blob/master/NeoML/docs/ru/API/NN/LossLayers/CrossEntropyLossLayer.md)